components/brave_ads/browser/search_result_ad/search_result_ad_aliases.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_ADS_BROWSER_SEARCH_RESULT_AD_SEARCH_RESULT_AD_ALIASES_H_
+#define BRAVE_COMPONENTS_BRAVE_ADS_BROWSER_SEARCH_RESULT_AD_SEARCH_RESULT_AD_ALIASES_H_
+
+#include <vector>
+
+#include "brave/vendor/bat-native-ads/include/bat/ads/public/interfaces/ads.mojom.h"
+
+namespace brave_ads {
+
+using SearchResultAdsList = std::vector<ads::mojom::SearchResultAdPtr>;
```

Comment:
please move this to search_result_ad_handler

Reply:
done